src/Components/ClockComment.js

```diff
@@ -0,0 +1,14 @@
+import { Row } from 'react-bootstrap';
+
+/** function generating the text to be displayed above the clock
+ * @returns {Node} representing component
+ */
+function ClockComment() {
+  return (
+    <Row xs={1} className="justify-content-center m-auto">
+      <p className="fs-5 fst-italic text-center">{props.text}</p>
```

Comment:
@tepeszCodes , where is the `props` parameter coming from?

Reply:
My bad, fixed it